code/items/perks/Test1.cs

```diff
@@ -9,6 +11,33 @@ public Test1() : base()
         {
 
         }
+        public override void OnEquip()
+        {
+            base.OnEquip();
+
+            VertexBuffer vb = new VertexBuffer(); // this should be called vertex builder or something
+            vb.Init(true);
+            vb.AddCube(Vector3.Zero, Vector3.One * 50.0f, Rotation.Identity);
+
+            var mesh = new Mesh(Material.Load("materials/default/vertex_color.vmat"));
+            mesh.CreateBuffers(vb);
+            mesh.SetBounds(Vector3.One * -25, Vector3.One * 25);
+
+            var model = new ModelBuilder()
+            .AddMesh(mesh)
+            .AddCollisionBox(Vector3.One * 25.0f)
+            .Create();
+
+            AnimEntity entity = new AnimEntity();
+            entity.SetModel(model);
+            entity.SetupPhysicsFromModel(PhysicsMotionType.Static);
+            entity.GlowActive = true;
+            entity.GlowColor = Color.Red;
+            entity.GlowState = GlowStates.GlowStateOn;
+            entity.GlowDistanceStart = 1;
+            entity.GlowDistanceEnd = 10000;
+            entity.Position = Owner.Position + Owner.EyeRot.Forward * 100f;
```

Comment:
If you find any of these values useful to define as constants, please do,

ex. `GLOW_DISTANCE_START = 1` // probably a better name

Just whenever someone needs to either modify these values in the future, or re-write them.

Reply:
Test1 is just a test for ingame things, nothing that has any function or should be kept, so please ignore this :D